frontend/src/components/common/table/SortMenu.tsx

```diff
@@ -78,8 +79,8 @@ const SortMenu = ({
           <>
             <PopoverTrigger>
               <Button
-                leftIcon={<FilterOptionsIcon />}
-                minWidth="5%"
+                leftIcon={<SortOptionsIcon />}
```

Comment:
For this change I noticed that the icon we're currently using doesn't match the one that appears in all of the designs. I updated the icon to reflect the designs, but please let me know if this decision was intentional and I can revert the change.

Reply:
i'm fine with updating the icon! but is `FilterOptionsIcon` used elsewhere? if not, maybe we can remove from our icons page?

Reply:
It is not used elsewhere, but there are a lot of icons that aren't currently used on that page which we kept (I assume to make it easier to start components with those icons. If we don't already have a ticket, my preference would be to make a ticket to track removal of ALL the unused icons at the end of the project. Thoughts?

Reply:
makes sense to me!